sdk/jdbc/azure-identity-providers-jdbc-postgresql/src/main/java/module-info.java

```diff
@@ -0,0 +1,11 @@
+// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+module com.azure.identity.providers.postgresql {
+    requires com.azure.identity;
+    requires com.azure.identity.providers.core;
+    requires org.postgresql.jdbc;
+    requires java.sql;
```

Comment:
Why do we need this here, but not in the MySQL one?

Reply:
Did you test with a JPMS application?

Reply:
> 

I think the reason is that [PSQLException](https://github.com/Azure/azure-sdk-for-java/blob/644f8356ea30fde64c3ad13a18d242557f7f80da/sdk/jdbc/azure-identity-providers-jdbc-postgresql/src/main/java/com/azure/identity/providers/postgresql/AzureIdentityPostgresqlAuthenticationPlugin.java#L9) extends java.sql.SQLException in postgre, but MySQL doesn't use it.

Reply:
> Why do we need this here, but not in the MySQL one?

I think the reason is that [PSQLException](https://github.com/Azure/azure-sdk-for-java/blob/644f8356ea30fde64c3ad13a18d242557f7f80da/sdk/jdbc/azure-identity-providers-jdbc-postgresql/src/main/java/com/azure/identity/providers/postgresql/AzureIdentityPostgresqlAuthenticationPlugin.java#L9) extends java.sql.SQLException in postgre, but MySQL doesn't use it.